packages/cashier/src/pages/payment-agent/payment-agent-detail/payment-agent-detail.tsx

```diff
@@ -10,7 +10,7 @@ type TPaymentAgentDetail = {
     has_red_color?: boolean;
     icon?: string;
     is_link?: boolean;
-    title?: string;
+    title?: string | React.ReactNode;
```

Comment:
`string | React.ReactNode` is basically just `React.ReactNode` because `string` is a variation of `React.ReactNode`
`string | React.ReactNode` is good though because it's more explicit

Reply:
"Explicit is better than implicit." ;););)